lib/jmap/mail/email/email.dart

```diff
@@ -120,6 +122,7 @@ class Email with EquatableMixin {
       bodyValues: (json['bodyValues'] as Map<String, dynamic>?)?.map((key, value) => EmailBodyValueConverter().parseEntry(key, value)),
       headerUserAgent: IndividualHeaderIdentifierNullableConverter().parseEntry(IndividualHeaderIdentifier.headerUserAgent.value, json[IndividualHeaderIdentifier.headerUserAgent.value] as String?),
       headerMdn: IndividualHeaderIdentifierNullableConverter().parseEntry(IndividualHeaderIdentifier.headerMdn.value, json[IndividualHeaderIdentifier.headerMdn.value] as String?),
+      headerCalendarEvent: IndividualHeaderIdentifierNullableConverter().parseEntry(IndividualHeaderIdentifier.headerCalendarEvent.value, json[IndividualHeaderIdentifier.headerCalendarEvent.value] as String?),
```

Comment:
because `calendar` is not a feature of `core` jmap, how to add it only when we need?

Reply:
> because `calendar` is not a feature of `core` jmap, how to add it only when we need?

Just add it to `properties` of `Email/get` and it will return this header value.